File: app-mvp/src/main/java/com/xuie/imaginaryandroid/gui/gank/meizhi/MeiZhiAdapter.java
package xuk.imaginary.gui.gank.meizhi;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import xuk.imaginary.R;
import xuk.imaginary.data.BaseBean;
import xuk.imaginary.common.GlideUtils;

import java.util.List;

/**
 * @author xuie
 * @date 17-7-5
 */
public class MeiZhiAdapter extends BaseQuickAdapter<BaseBean, BaseViewHolder> {
    private static final String TAG = "MeiZhiAdapter";

    public MeiZhiAdapter(List<BaseBean> meiZhis) {
        super(R.layout.item_fragment_meizhi, meiZhis);
    }

    @Override
    protected void convert(BaseViewHolder helper, BaseBean item) {
        helper.setText(R.id.date, item.getDesc());
        GlideUtils.loadImageMeizhi(mContext, item.getUrl(), helper.getView(R.id.meizhi));
    }
}

File: app-mvp-kotlin/src/main/java/xuk/imaginary/gui/gank/meizhi/MeiZhiAdapter.kt
package xuk.imaginary.gui.gank.meizhi

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder
import xuk.imaginary.R
import xuk.imaginary.data.BaseBean
import xuk.imaginary.util.GlideUtils

/**
 * Created by xuie on 17-7-5.
 */

class MeiZhiAdapter(meiZhis: List<BaseBean>?) : BaseQuickAdapter<BaseBean, BaseViewHolder>(R.layout.item_fragment_meizhi, meiZhis) {

  override fun convert(helper: BaseViewHolder, item: BaseBean) {
    helper.setText(R.id.date, item.desc)
    GlideUtils.loadImageMeizhi(mContext, item.url, helper.getView(R.id.meizhi))
  }

  companion object {
    private val TAG = "MeiZhiAdapter"
  }
}
